tests/unit/login_test.go

```diff
@@ -24,15 +25,36 @@ var credsOutput = fmt.Sprintf(constants.CredentialsExport,
 var testCases = []struct {
 	name          string
 	leaseID       string
-	openBrowser   bool
-	printCreds    bool
-	profile       string
+	opts          *service2.LeaseLoginOptions
 	expectedOut   string
 	isWeberCalled bool
 }{
-	{"GIVEN no flags THEN do not print anything", "doesntMatter", false, false, "default", "", false},
-	{"GIVEN openBrowser THEN Weber should open browser", "doesntMatter", true, false, "default", "Opening AWS Console in Web Browser", true},
-	{"GIVEN printCreds THEN Weber should open browser", "doesntMatter", false, true, "default", credsOutput, false},
+	{
```

Comment:
just restructured these to use the new `LeaseLoginOptions` struct (no other changes to this test)